### PR TITLE
Uploading our file structure to start coding in our Test Branch

### PR DESCRIPTION
**Pull request by htdguide**

### CODE REVIEW

**Review by gnijen — APPROVED** [review]
The main page and the log in looks good. Great work with making the login data working and the user name comes up in at the top of the home page. Good work overall

File: studio3code/src/LoginPage.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      navigate('/member');
+    } else {
+      alert('Please try again.');
+    }
+  };
+
+  return (
+    <section>
+      <h2>Login</h2>
+      <div>
```
Comment (abhistyalish2802) [inline studio3code/src/LoginPage.js:21]: i Have to add 2-3 headings in this

**Review by Sahil4353 — COMMENTED** [review]
I want to add 2 username and password

File: studio3code/src/index.css [hunk 2]
```diff
@@ -1,12 +1,11 @@
-/* src/styles.css */
 body {
   font-family: 'Arial', sans-serif;
   margin: 0;
   padding: 0;
 }
```
Comment (abhistyalish2802) [inline studio3code/src/index.css:6]: want to work more on keyframes and media

**Review by abhistyalish2802 — COMMENTED** [review]
keyframes and media in css

**Comment by gnijen** [comment]
Overall, great job on your contribution! Your code demonstrates a solid understanding of both the problem domain and software development. Your code is well-structured and easy to follow. Your code aligns well with our team's coding standards.

**Pull request merged**